qiskit/circuit/quantumcircuit.py

```diff
@@ -604,6 +604,12 @@ def draw(self, scale=0.7, filename=None, style=None, output=None,
             idle_wires (bool): Include idle wires. Default is True.
             with_layout (bool): Include layout information, with labels on the physical
                 layout. Default is True.
+            fold (int): Sets pagination. It can be disabled using -1.
+                In `text`, sets the length of the lines. This useful when the
+                drawing does not fit in the console. If None (default), it will try to
+                guess the console width using `shutil.get_terminal_size()`. However, if
+                running in jupyter, the default line length is set to 80 characters.
+                In `mpl` is the amount of operations before folding. Default is 25.
```

Comment:
It might be worth saying explicitly that for `latex` and `latex_source` this does nothing